### DISCUSSION TITLE
[github/explore] Bug when parsing unordered lists in topic Markdown

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#86655] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area
Bug

### Body

I was wondering if the list parsing was intentional:
| Actual | Expected |
|-|-|
| ![grafik](https://github.com/github/explore/assets/11225821/9714b28b-267b-49be-b2cc-28816ca8e63e) | ![grafik](https://github.com/github/explore/assets/11225821/a3b4a4cc-d1d9-4d5b-a81b-0c81d6ead1ba) |
| ![grafik](https://github.com/github/explore/assets/11225821/4bfc0fe0-b314-41e1-a48d-c0202eb0ac9f) | ![grafik](https://github.com/github/explore/assets/11225821/c96d65e1-37c4-41e1-9f0f-5189c45d69e2) |

But if you look at [the feed](https://explore-feed.github.com/feed.json), it's all inside one `<ul>`, however, delimited by `\n`:
```html
<p><strong>Backbone.js</strong> supplies structure to JavaScript-heavy applications by providing models with key-value binding and custom events, collections with a rich API of enumerable functions, views with declarative event handling, and connects it all to your existing application over a RESTful JSON interface. Backbone.js was originally extracted from the Rails application <a href="http://www.documentcloud.org/">DocumentCloud</a>. Philosophically, Backbone is an attempt to discover the minimal set of data-structuring (models and collections) and user interface (views and URLs) primitives that are generally useful when building web applications with JavaScript. Backbone is a library, not a framework. Synchronous events are used as the fundamental building block over constantly polling data. The main pars of Backbone are:</p>\n<ul>\n  <li>Events</li>\n  <li>Models – Wraps a row of data in business logic.</li>\n  <li>Collections – A group of models on the client-side, with sorting/filtering/aggregation logic.</li>\n  <li>Router (+ History)</li>\n  <li>Views (+ Client-side Templates) – A logical, re-usable piece of UI. Often, but not always, associated with a model.</li>\n  <li>Sync – Synchronization between frontend and REST API backend</li>\n</ul>\n
```

As far as I can see, that part is not open source, so I can't fix that myself and create a PR, right?

<small>(Originated here: https://github.com/github/explore/pull/3980#issuecomment-1762414819)</small>

**Comment by Naimulnobel** [comment]
It seems like you're experiencing an issue with how list items are being parsed and displayed. From your description, it looks like the list items are being displayed as separate items when they are actually part of the same unordered list (`<ul>`).

This could be due to how the HTML is being parsed and rendered. If the parser is not correctly handling the newline characters (`\n`) between list items, it could cause them to be displayed as separate lists.

As for your question about contributing a fix, it depends on the specific project and its contribution guidelines. Some projects are open source and welcome contributions from the community, while others are not.

If the project is open source, you would typically fork the repository, make your changes in your fork, and then submit a pull request to the original repository. The maintainers of the project would then review your changes and decide whether to merge them into the main project.

If the project is not open source, or if the part of the codebase you want to change is not available, you might not be able to submit a fix yourself. In that case, the best course of action would be to report the issue to the maintainers of the project. Be sure to include as much detail as possible, including a clear description of the issue, steps to reproduce it, and any relevant screenshots or error messages.

**Reply by shaedrich** [reply]
It would be so good if people were still able to formulate their own answers instead of using ChatGPT to just paraphrase the question
